src/fuse_scan.cpp

```diff
@@ -160,12 +165,15 @@ void FusedScan::fusedScanCallback(const sensor_msgs::LaserScan::ConstPtr& scan_1
 
     //convert scan to pointcloud
     cloud_fuse_.points.clear();
-    sensor_msgs::PointCloud cloud_msg;
+    sensor_msgs::PointCloud cloud_msg, cloud_filtered_;
```

Comment:
cloud_filtered_ cannot have a leading underscore, since it is not a private variable